apps/ios/package.json

```diff
@@ -20,20 +20,21 @@
   },
   "dependencies": {
     "@fluentui-react-native/tester": "^0.14.0",
-    "react": "16.11.0",
-    "react-native": "0.62.2",
-    "react-native-test-app": "^0.3.2"
+    "react": "16.13.1",
+    "react-native": "0.63.4",
```

Comment:
Just noticed this now but wouldn't we want iOS to still use react-native-macos so it has all our extra fixes we made that haven't made it upstream yet

Reply:
I thought about this earlier too, from when we first moved the test apps to per platform. At the time I thought it made more sense for a public facing repo to prefer the facebook official React Native if possible. I can't really think of a benefit now besides seeing how FURN operates without our extra fixes in + it's easier to update the iOS app to the latest RN without waiting for react-native-macos to catch up. 

I could go either way, but would also do it in a separate change if we decided to go back to react-native-macos

Reply:
I think it's more important that this repro presents a consistent baseline to our clients. If iOS is shipping with a different version and therefore different fixes than macOS that's probably not a good message to send. Agree it can be fixed outside this PR, but should probably get addressed soon